docs/tutorial/t-relate-kafka.md

```diff
@@ -21,9 +23,9 @@ Located charm "data-integrator" in charm-hub, revision 11
 Deploying "data-integrator" from charm-hub charm "data-integrator", revision 11 in channel stable on jammy
 ```
 
-### Relate to Kafka
+### Relate to Apache Kafka
 
-Now that the Database Integrator Charm has been set up, we can relate it to Kafka. This will automatically create a username, password, and database for the Database Integrator Charm. Relate the two applications with:
+Now that the Database Integrator Charm has been set up, we can relate it to Apache Kafka. This will automatically create a username, password, and database for the Database Integrator Charm. Relate the two applications with:
```

Comment:
```suggestion
Now that the Database Integrator Charm has been set up, we can relate it to Charmed Apache Kafka. This will automatically create a username, password, and database for the Database Integrator Charm. Relate the two applications with:
```